app/javascript/alchemy_admin/picture_selector.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Multiple picture select handler for the picture archive.
+ */
+export default function PictureSelector() {
+  const $selected_item_tools = $(".selected_item_tools")
+  const $picture_selects = $(".picture_tool.select input")
+
+  $picture_selects.on("change", function () {
```

Comment:
We have [the events module under utils](https://github.com/AlchemyCMS/alchemy_cms/blob/main/app/javascript/alchemy_admin/utils/events.js) that we can use to get rid of jQuery.on here